packages/TorneloScoresheet/src/pages/GraphicalRecording/GraphicalRecording.tsx

```diff
@@ -52,11 +52,13 @@ const GraphicalRecording: React.FC = () => {
   const [pgn, setPgn] = useState('');
   const [, showError] = useError();
   const [showEndGame, setShowEndGame] = useState(false);
-  const [showSignature, setShowSignature] = useState(false);
+  const [showFirstSignature, setShowFirstSignature] = useState(false);
+  const [showSecondSignature, setShowSecondSignature] = useState(false);
   const goToEndGame = graphicalRecordingState?.[1].goToEndGame;
   const [selectedWinner, setSelectedWinner] = useState<
     undefined | Player | null
   >(undefined);
+  const [playerSignatures, setPlayerSignatures] = useState<string[]>([]);
```

Comment:
I think it's better to have a Record<PlayerColor, string|undefined> here too, it will be undefined untill a signiture is stored.